<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_font">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/Servicios.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QFrame" name="destinos">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>29</y>
     <width>485</width>
     <height>456</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
border: 1px solid #c7c7c7;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_head">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>486</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
background-color:rgb(255, 0, 0);
font: 75 20pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_vuelta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Vuelta 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_estudiantes_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 16pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>TOTAL A LIQUIDAR             $</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_a_liquidar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 15pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>21.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_operador">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0;
font: 14pt &quot;Roboto&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Op: Juan PÃ©rez</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_fin">
   <property name="geometry">
    <rect>
     <x>621</x>
     <y>375</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/cerrarTurno.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_user">
   <property name="geometry">
    <rect>
     <x>548</x>
     <y>110</y>
     <width>203</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/User/user/generico.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_cancel">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>385</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/no.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_cambiar_ruta">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Imagenes/cambiarRuta.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_vigencia_tarjeta">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>VIGENCIA: 31/DIC/2022 23:59</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
